plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/messages.properties

```diff
@@ -1,2 +1,9 @@
 NEW_SERVER_DIALOG_PORT=Server port:
 NEW_SERVER_DIALOG_INVALID_PORT_VALUE=Port must be between 0 and 65535.
+
+CREATE_APP_ENGINE_RUNTIME_WIZARD_DESCRIPTION=The App Engine Standard runtime requires the Google Cloud SDK
+CREATE_APP_ENGINE_RUNTIME_WIZARD_TITLE=App Engine Standard Runtime
+RUNTIME_WIZARD_CLOUD_SDK_FOUND=The location of the Cloud SDK has been identified as {0}.
+RUNTIME_WIZARD_CLOUD_SDK_NOT_FOUND=Cannot find the Google Cloud SDK. Click 'Update SDK Location' to configure it's location.
+RUNTIME_WIZARD_OPEN_CLOUD_SDK_DIALOG_MESSAGE=Do you want to cancel the creation of an App Engine Standard runtime and update the location of the Google Cloud SDk?
```

Comment:
Unclear on why we need this. Has the user not already clicked "Update SDK Location'?